src/hooks/useSnackMessage.ts

```diff
@@ -23,81 +28,57 @@ export interface UseSnackMessageReturn {
     snackError: (snackInputs: SnackInputs) => void;
     snackWarning: (snackInputs: SnackInputs) => void;
     snackInfo: (snackInputs: SnackInputs) => void;
+    closeSnackbar: (key?: SnackbarKey) => void;
```

Comment:
Maybe `typeof closeSnackbar` as we just forward notistack method

Reply:
done